test/integration/test-example-validation.js

```diff
@@ -74,6 +75,8 @@ describe('example', function() {
     // contains more specific script identifier.
     /The tag \'script\' is disallowed/,
     /\'amp-brid-player\' is disallowed/,
+    /invalid value \'amp-springboard-player\'/,
```

Comment:
The `/invalid value \'amp-springboard-player\'/,` is not needed here - please remove.